cibuild.cmd

```diff
@@ -79,14 +80,20 @@ if "%SYNC_XLF%"=="true" (
     set SYNC_XLF_ARGUMENT="/p:SyncXlf=true"
 )
 
+:: Full Framework MSBuild does not have the new built-in property MSBuildRuntimeType
+set RUNTIMETYPE_ARGUMENT=
+if "%TARGET%"=="Full" (
```

Comment:
We use installed MSBuild for the first build on Desktop ("Full" now, I guess 🙂). It's not unreasonable to do so for CoreCLR as well, though it is nice to get a) coverage on .NET Core MSBuild b) uniformity between first- and second-phase builds and c) a no-install-required experience.